api/src/main/resources/workbench-api.yaml

```diff
@@ -5407,6 +5413,10 @@ definitions:
 
 '
         type: string
+  WorkspaceResourcesRequest:
```

Comment:
nit: for future-proofness, I might put your array in a member so you can add other kinds of criteria to this request later if needed.